src/Cronofy/AddToCalendarRequestBuilder.cs

```diff
@@ -130,14 +141,98 @@ public AddToCalendarRequestBuilder UpsertEventRequest(UpsertEventRequest upsertE
             return this;
         }
 
+        /// <summary>
+        /// Sets the availability details of the request.
+        /// </summary>
+        /// <param name="availabilityRequestBuilder">
+        /// The availability details builder for the request, must not be null.
+        /// </param>
+        /// <returns>
+        /// A reference to the <see cref="AddToCalendarRequestBuilder"/> 
+        /// </returns>
+        public AddToCalendarRequestBuilder AvailabilityRequestBuilder(IBuilder<AvailabilityRequest> availabilityRequestBuilder)
+        {
+            Preconditions.NotNull("availabilityRequestBuilder", availabilityRequestBuilder);
+
+            this.availabilityRequestBuilder = availabilityRequestBuilder;
+
+            return this;
+        }
+
+        /// <summary>
+        /// Sets the availability details of the request.
+        /// </summary>
+        /// <param name="availabilityRequest">
+        /// The event details for the request, must not be null.
+        /// </param>
+        /// <returns>
+        /// A reference to the <see cref="AddToCalendarRequestBuilder"/>.
+        /// </returns>
+        public AddToCalendarRequestBuilder AvailabilityRequest(AvailabilityRequest availabilityRequest)
+        {
+            Preconditions.NotNull("availability", availabilityRequest);
+
+            this.availabilityRequestBuilder = Builder.Wrap(availabilityRequest);
+
+            return this;
+        }
+
+        /// <summary>
+        /// Sets the target calendar details of the request.
+        /// </summary>
+        /// <param name="sub">
+        /// The sub for the target calendar.
+        /// </param>
+        /// <param name="calendarId">
+        /// The target calendar's id.
+        /// </param>
+        /// <returns>
+        /// A reference to the <see cref="AddToCalendarRequestBuilder"/>.
+        /// </returns>
+        public AddToCalendarRequestBuilder TargetCalendars(string sub, string calendarId)
+        {
+            Preconditions.NotBlank("sub", sub);
+            Preconditions.NotBlank("calendarId", calendarId);
+
+            return TargetCalendars(new[] { new AddToCalendarRequest.TargetCalendar { 
```

Comment:
I would expect this to be adding to the set of target calendars rather than replacing if you think about builder usage it would be something like 

```csharp
var foo = new FooBuilder()
    .TargetCalendars("Harry", "cal_123")
    .TargetCalendars("Stephen", "cal_456")
    .Build(); // sucks to be your calendar.
```